Plots/Trajectories/NCL_traj_1.py

```diff
@@ -0,0 +1,97 @@
+"""
+NCL_traj_1.py
+===============
+This script illustrates the following concepts:
+   - Plotting a simple trajectory plot
+   - Plotting multiple trajectories in different colors
+   - Plotting every nth time step in a trajectory
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/traj_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/traj_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import xarray as xr
+import geocat.datafiles as gdf
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+import numpy as np
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get('netcdf_files/traj_data.nc'))
+sdata = ds.get('sdata')
+
+
+###############################################################################
+# Define helper function to plot every fourth timestep:
+
+def plot4thTimestep1(nparrayy, nparrayx):
+
+    for x in range(0, len(nparrayx)):
+
+        # Plot green starting point of each trajectory
+        if x == 0:
+            y, x = nparrayy[x], nparrayx[x]
+            plt.scatter(x, y, color='green', s=4)
+
+        # Plot every fourth timestamp
+        if (x + 1) % 4 == 0:
+            y, x = nparrayy[x], nparrayx[x]
+            plt.scatter(x, y, color='black', s=4)
+
+
+###############################################################################
+# Plot:
+
+# Initialize axes
+ax = plt.axes(projection=ccrs.PlateCarree())
+ax.set_extent([-75, -25, -60, -20], crs=None)
+
+# Set title and subtitle
+plt.suptitle('Example of a Trajectory Plot')
+plt.title('markers every 4th timestep', fontsize=10, pad=10)
+
+# Set land feature and change color to 'lightgrey'
+# See link for extensive list of colors:
+# https://matplotlib.org/3.1.0/gallery/color/named_colors.html
+ax.add_feature(cfeature.LAND, color='lightgrey')
+
+# Set formatted axes ticks (with degree symbol and direction)
```

Comment:
How about replacing Lines 66 through 76 with the following convenience functions from geocat.viz utility package?

```
# Use geocat.viz.util convenience function to make plots look like NCL plots by using latitude, longitude tick labels
gvutil.add_lat_lon_ticklabels(ax)

# Use geocat.viz.util convenience function to set axes tick values
gvutil.set_axes_limits_and_ticks(ax, xticks=np.linspace(-70, -30, 5), yticks=np.linspace(-60, -20, 5))
```

To use the above functions, don't forget to add following to package import block:

`from geocat.viz import util as gvutil`